Library/Homebrew/test/support/fixtures/cask/Casks/with-uninstall-preflight.rb

```diff
@@ -0,0 +1,11 @@
+cask 'with-uninstall-preflight' do
+  version '1.2.3'
+  sha256 '8c62a2b791cf5f0da6066a0a4b6e85f62949cd60975da062df44adf887f4370b'
+
+  url "file://#{TEST_FIXTURE_DIR}/cask/MyFancyPkg.zip"
+  homepage 'http://example.com/fancy-pkg'
+
+  pkg 'MyFancyPkg/Fancy.pkg'
+
+  uninstall_preflight do end
```

Comment:
Put the `end` on a separate line for these.

Reply:
👍